### ISSUE TITLE
[Warlock] Unstable affliction dispell damage takes victims spell power

### ISSUE THREAD

**Issue by Jumpka** [ccshiro/cc-buglist#1892]
The spell damage of warlock doesn't affect the dispell damage at all, it's all on the TARGET of the UA that is being dispelled.
Steps to reproduce:
1. Cast Unstable affliction on target with 0 spell damage and dispell it
2. Cast Unstable affliction on target with high spell damage and dispell it

Note: Also the yellow damage number pops up for the guy affected by UA, making it look like he was damaging his own dispeller

**Comment by shadowraider** [comment]
also check if the dispel dmg is increased with Shadow Mastery talent

**Comment by Jumpka** [comment]
Can't test it now, but I'm pretty sure the bug is simple and that it is increased only by damage increasing talents of the character from which the debuff is dispelled.
Basically it like if the dispell damage was casted by the dispelled target.

**Comment by Summoning01** [comment]
results from a bit of testing:

_attacker_: affliction warlock with 5/5 shadow mastery (1184 bonus spell dmg)

_target 1_: paladin (31 bonus spell dmg)
- ua dispel dmg: **1630** (1575 + 31 \* 1.8 = 1630.8)
- ua dot: **2712** ((1050 + 1184 \* 1.2) \* 1.1 = 2717.88)

_target 2_: paladin (462 bonus spell dmg)
- ua dispel dmg: **2406**
- ua dot: **2712**

_target 3_: affliction warlock (also with 5/5 SM, 1134 bonus spell dmg)
- ua dispel dmg (spellstone): **3616**
- ua dispel dmg (devour magic): **3616**
- ua dot: **2712**

**tl;dr** - yes, target's bonus damage is used instead of attacker's and neither attacker's nor target's (if lock) shadow mastery is part of the ua dispel damage equation.
